test/integration/config_error_event_test.go

```diff
@@ -0,0 +1,105 @@
+//go:build integration_tests
+// +build integration_tests
+
+package integration
+
+import (
+	"context"
+	"testing"
+
+	"github.com/kong/kubernetes-testing-framework/pkg/clusters"
+	"github.com/kong/kubernetes-testing-framework/pkg/utils/kubernetes/generators"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	"github.com/kong/kubernetes-ingress-controller/v2/internal/annotations"
+	"github.com/kong/kubernetes-ingress-controller/v2/internal/dataplane"
+	"github.com/kong/kubernetes-ingress-controller/v2/internal/versions"
+	"github.com/kong/kubernetes-ingress-controller/v2/test"
+	"github.com/kong/kubernetes-ingress-controller/v2/test/consts"
+	"github.com/kong/kubernetes-ingress-controller/v2/test/internal/helpers"
+	"github.com/kong/kubernetes-ingress-controller/v2/test/internal/testenv"
+)
+
+func TestConfigErrorEventGeneration(t *testing.T) {
+	// this test is NOT parallel. the broken configuration prevents all updates and will break unrelated tests
+	if testenv.DBMode() != "off" {
+		t.Skip("config errors are only supported on DB-less mode")
+	}
+	if versions.GetKongVersion().MajorMinorOnly().LTE(versions.FlattenedErrorCutoff) {
+		t.Skipf("kong version is %s <= 3.2, skipping testing config error parsing", versions.GetKongVersion().MajorMinorOnly().String())
```

Comment:
should be `<=3.1`?

Reply:
I changed the check a bit and forgot to update the log. Now logging `<3.2` since, while not the exact check, is basically what we want to communicate.